src/test/java/com/klarna/hiverunner/ParquetInsertionTest.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (C) 2013-2020 Klarna AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.klarna.hiverunner;
+
+import com.klarna.hiverunner.annotations.HiveSQL;
+import com.klarna.hiverunner.annotations.HiveSetupScript;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Arrays;
+
+@RunWith(StandaloneHiveRunner.class)
+public class ParquetInsertionTest {
+    /***
```

Comment:
Can you move this up to be a class level comment please?

Reply:
Ah yeah I've clearly been writing too much Python recently, where docstrings go after the start of the class/ function they refer to. Will change, thanks.